extension/pages/settings/settings.html

```diff
@@ -876,7 +876,7 @@ <h2>
 						</div>
 					</section>
 					<section name="companies">
-						<div class="header">Job Specials</div>

```

Comment:
This header was specifically chosen to make it more clear what those settings are about. Instead of changing it, you should make a new header.